manifests/init.pp

```diff
@@ -73,28 +67,27 @@
     validate_absolute_path($install_dir)
   }
 
```

Comment:
I wonder if it'd be worth adding a check here to error if `$::aio_agent_version` is undefined (i.e. error if we're running on a pre-Puppet 4 node).

Reply:
My Q. about `$::aio_agent_version` is true, so my other Q. about whether it's worth erroring if that variable is undefined still stands.

Reply:
I think so, yes. Major things depend on it being set.

Reply:
I'll push up another commit to add this capability then.